_posts/2019-07-29-plonk-stack.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Introducing the PLONK Stack for Cloud Native Developers"
+description: You've heard of LAMP, JAM, and MEAN, but what's the PLONK stack, and why should you be considering it for your Cloud Native Applications?
```

Comment:
I'd consider:
`You've heard of LAMP, JAM, and MEAN, so here's the PLONK stack, and why you should be considering it for your Cloud Native Applications.`